### PR TITLE
Add kubectl-neat version command

### LINKED ISSUE #63: Suggestion: Add version flag
It's currently not possible to request the version of kubectl-neat. I suggest an build in --version option to display the current version.

### PR DESCRIPTION
**Pull request by amirschw**
Add "version" command to kubectl-neat.

Closes #63

```bash
~ $ kubectl-neat version
kubectl-neat version: v2.0.3-next

~ $ kubectl-neat version --help
Print the version of kubectl-neat

Usage:
  kubectl-neat version [flags]

Flags:
  -h, --help   help for version

Global Flags:
  -o, --output string   output format: yaml or json (default "yaml")

~ $ kubectl-neat --help
Usage:
  kubectl-neat [flags]
  kubectl-neat [command]

Examples:
kubectl get pod mypod -o yaml | kubectl neat
kubectl get pod mypod -oyaml | kubectl neat -o json
kubectl neat -f - <./my-pod.json
kubectl neat -f ./my-pod.json
kubectl neat -f ./my-pod.json --output yaml

Available Commands:
  get         
  help        Help about any command
  version     Print kubectl-neat version

Flags:
  -f, --file string     file path to neat, or - to read from stdin (default "-")
  -h, --help            help for kubectl-neat
  -o, --output string   output format: yaml or json (default "yaml")

Use "kubectl-neat [command] --help" for more information about a command.
```

### CODE REVIEW

File: cmd/cmd.go [hunk 1]
```diff
@@ -127,6 +128,15 @@ kubectl neat get -- svc -n default myservice --output json`,
 	},
 }
 
+var versionCmd = &cobra.Command{
+	Use:   "version",
+	Short: "Print kubectl-neat version",
+	Long:  "Print the version of kubectl-neat",
+	Run: func(cmd *cobra.Command, args []string) {
+		fmt.Printf("kubectl-neat version: %s\n", Version)
```
Comment (amirschw) [inline cmd/cmd.go:136]: Hi @itaysk, hope you're well :)

I started off with this basic implementation that only prints the version. Let me know if you would like to expand it by displaying other information like git commit or build date, and then perhaps also introduce a `--short` flag.

Also note that I am currently ignoring the persistent `--output` flag, as I couldn't find a good way to disable a global flag from a specific command. The version can of course be printed as JSON or YAML if you don't find it an overkill.

**Comment by itaysk** [comment]
Thanks @amirschw ! Nice to hear from you. 
Looks good to me. My only comment is to maybe add the Version variable to the existing cmd.go file instead of creating a new file just for one variable. WDYT?

**Comment by amirschw** [comment]
You too @itaysk! and thanks for kubectl-neat, we find it very useful here.
As for the comment - absolutely, I moved the variable to the cmd.go file. Should I squash the commits?

**Review by itaysk — APPROVED** [review]

**Comment by itaysk** [comment]
> Should I squash the commits?

I can do that as part of the merge

**Pull request merged**